### ISSUE TITLE
Save/Restore configuration not possible with onRefresh function

### ISSUE THREAD

**Issue by crestAT** [nicolaskruchten/pivottable#945]
First of all THANKS for this great plugin!

I use the Save/Restore configuration as in the example page and it's working but the problem for me is that I (have to) use onRefresh, which is not saved (I suppose because it's a function).

So if I reload the configuration the onRefresh mechanism is no longer available.

The question is, is it possible and if yes, how, to add the onRefresh code after reloading?

Many thanks in advance!

**Comment by nicolaskruchten** [comment]
Yes, you cannot serialize functions, so you'll need to re-add the `onRefresh` function to the saved config before passing it back in.
